TypeCobol/Compiler/Sql/Model/SqlObject.cs

```diff
@@ -18,6 +19,43 @@ public bool AcceptVisitor(ISqlVisitor visitor)
             visitor.EndSqlObject(this);
             return continueVisit;
         }
+        public virtual void Dump(TextWriter output, int indentLevel)
+        {
+            string indent = new string(' ', 2 * indentLevel);
+            output.Write(indent);
+            output.WriteLine(GetType().FullName);
+        }
+
+        protected static void DumpProperty(TextWriter output, string name, object value, int indentLevel)
+        {
+            string indent = new string(' ', 2 * indentLevel);
+            output.Write($"{indent}- {name} = ");
+            if (value == null)
+            {
+                output.WriteLine("<NULL>");
+            }
+            else if (value is SqlObject sqlObject)
+            {
+                output.WriteLine(sqlObject.GetType().Name);
+                sqlObject.Dump(output, indentLevel + 1);
+            }
+            else if (value is System.Collections.IEnumerable enumerable)
+            {
+                output.WriteLine("[");
+                int index = 0;
+                foreach (var item in enumerable)
+                {
+                    DumpProperty(output, name + index, item, indentLevel + 1);
```

Comment:
To get something like `table[0] =` in resulting text, you have to use `$"{name}[{index}]"` instead of `name + index`